kernel/core/net/net_client.c

```
@@ -502,15 +514,37 @@ fl_net_client_dispatch_frame(fl_net_client_t *client, uint8_t opcode,
                    ? FL_NET_SERVER_EVENT_HOST_PROMOTE
                    : FL_NET_SERVER_EVENT_HOST_REDIRECT;
     }
+    if (opcode == (uint8_t)FL_NET_SESSION_OP_FILE_OFFER) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)